contracts/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+...
+
+## 0.5.1 - ..........
```

Comment:
add today's date here?

Reply:
Nice catch, added in e44915c460e709977422a8cb5b80c9d0287c6f3a